src/forge/actors/replay_buffer.py

```diff
@@ -19,89 +21,111 @@
 logger.setLevel(logging.INFO)
 
 
+@dataclass
+class BufferEntry:
+    data: "Episode"
+    sample_count: int = 0
+
+
+def default_evict(
+    buffer: deque, policy_version: int, max_samples: int = None, max_age: int = None
+):
+    """Default buffer eviction policy"""
+    indices = []
+    for i, entry in enumerate(buffer):
+        if max_age and policy_version - entry.data.policy_version > max_age:
+            continue
+        if max_samples and entry.sample_count >= max_samples:
+            continue
+        indices.append(i)
+    return indices
+
+
+def default_sample(
```

Comment:
Rename this to the actual things they do lol.

``default_sample`` -> ``random_sample``